fossology/fossology
# feat(decision-dump): export and import IPRA agent's data

## Description

Export and import data from IPRA agent as well in `decisionexporter` and `decisionimporter`.

### Changes

1. Fetch data from `ipra` agent tables in `decisionexporter`
2. Import data for `ipra` agent tables in `decisionimporter`

## How to test

1. Run IPRA agent on an upload.
2. Export decision dump.
3. Check if dump contains data from `ipra`, `ipra_event` and `ipra_decision` tables.
4. Import the dump in another upload.
5. Check if data is imported for `ipra` as well.

<a href="https://gitpod.io/#https://github.com/fossology/fossology/pull/2387"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
